example/lib/develop/util/custom_colors.dart

```diff
@@ -0,0 +1,8 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+abstract class CustomColors {
```

Comment:
is it a good practice of keeping custom styles of the app?

Reply:
I'm not sure. I needed some place to store colors that I needed to use somewhere throughout the app (like for some view's background color) - what would be the better way to do that?

Reply:
Created a separate issue for that.